### PR TITLE
fix: add models and md5

### PR DESCRIPTION
**Pull request by ZiniuYu**
This pr adds open-clip model filenames and their corresponding md5
We now host a copy of open-clip models and convert them to onnx runtime

### CODE REVIEW

File: server/clip_server/model/openclip_model.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def __init__(self, name: str, device: str = 'cpu', jit: bool = False, **kwargs):
         if model_url:
             model_path = download_model(model_url, md5sum=md5sum)
             self._model = load_openai_model(model_path, device=device, jit=jit)
-            self._model_name = name
+            self._model_name = name.split('::')[0]
```
Comment (ZiniuYu) [inline server/clip_server/model/openclip_model.py:27]: @numb3r3 Check this

Comment (shan-mx) [reply server/clip_server/model/openclip_model.py:27]: > @numb3r3 Check this

should be split[0], I had fixed it in my pr

Comment (numb3r3) [reply server/clip_server/model/openclip_model.py:27]: I think the problem caused here can be addressed by another PR. So let's revert this change in this PR.

**Review by numb3r3 — CHANGES_REQUESTED** [review]

**Review by numb3r3 — APPROVED** [review]
LGTM

File: server/clip_server/model/openclip_model.py (hunk 1 above)
Comment (ZiniuYu) [reply server/clip_server/model/openclip_model.py:27]: we need this to pass ci

**Pull request merged**